README.md

```diff
@@ -9,6 +9,61 @@ This page is hosted on: https://members.realdevsquad.com/
 
 Wish to contribute? You can find a detailed guide [here](./CONTRIBUTING.md)!
 
+## How to start ?
+
+### Setup
+
+We use `npm` for managing packages.
+It is installed by default along with node.js.
+
+Please install `volta`
```

Comment:
Here, we are moving ahead with yarn
Ensure we change `npm` to `yarn`

Reply:
I'll update the changes